easybuild/easyconfigs/p/Perseus/Perseus-2.0.7.0-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'Tarball'
+
+name = 'Perseus'
+version = '2.0.7.0'
+
+homepage = 'https://maxquant.org/perseus'
+description = """The Perseus software platform supports biological and biomedical researchers
+    in interpreting protein quantification, interaction and post-translational modification data."""
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+# requires registration at https://maxquant.org/download_asset/perseus/latest
```

Comment:
we should use `download_instructions` since that will be more informative when the source tarball cannot be found

Reply:
That is a good improvement over the old easyconfig files. Could we use variable, which specifies the destination for the source file? Since if I specify it in a static text it will maybe not match.

Reply:
I don't think it's necessary to specify the destination. I think something along the lines of this should be sufficient:
```
download_instructions = """
To download the Perseus source tarball, register at:
    https://maxquant.org/download_asset/perseus/latest
You should receive a unique download URL via email.
"""
```